vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/StaffSearchModule.java

```diff
@@ -0,0 +1,37 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import java.util.List;
+import java.util.Map;
+
+public class StaffSearchModule {
+
+    private List<Module> module;
+
+    private Map<String, String> moduleFirstSlideFirstImage;
+
+    private Map<String, Boolean> moduleAlertMessage;
```

Comment:
bad variable names; second one should be at least plural; first one is unclear what that means.